docs/elasticsearch.md

```diff
@@ -0,0 +1,204 @@
+# Elasticsearch
```

Comment:
I just saw at the end of the PR that there is a unit file in the examples directory. It is weird to have things that belong together in different places. in `docs` there is this guide here, and in `example` there is the unit file. 😕 

I ignore the inconsistency of singular vs plural in the directory naming of docs vs example because this has nothing to do with this PR. 😒